rn-cli.config.js

```diff
@@ -19,22 +19,5 @@ module.exports = {
   extraNodeModules: {
     'react-native': __dirname,
   },
-  serializer: {
-    getModulesRunBeforeMainModule: () => [
-      require.resolve('./Libraries/Core/InitializeCore'),
-    ],
-    getPolyfills,
-  },
-  resolver: {
-    hasteImplModulePath: require.resolve('./jest/hasteImpl'),
```

Comment:
how does this affect out-of-tree platforms? cc @empyrical @rozele

Reply:
I think it shouldn't. This was added 2 weeks ago for me "as a workaround" and I reverted the format of config to what it's there already.

https://github.com/facebook/react-native/commit/1af390be19000b2756fc646e72f152dd383a0bb1#diff-41fdc10e1b5862877e4afa7e9176ee09

Reply:
Yeah, it looks fine for me in that regard. I can try and test it out later and see if everything still works just to double check